app/actions/views/account_notifications.js

```diff
@@ -21,9 +21,27 @@ export function handleUpdateUserNotifyProps(notifyProps) {
                 value: interval
             }];
 
-            await savePreferences(emailInterval)(dispatch, getState);
+            dispatch({
```

Comment:
actually after talking to the team we decided that we should not create separate actions, lets just convert the current ones to be optimistic